.codecov.yml

```diff
@@ -7,4 +7,5 @@ coverage:
         base: auto 
 comment:                  # this is a top-level key
   layout: "header, files, footer" # remove "new" from "header" and "footer"
-  hide_project_coverage: false # set to false
\ No newline at end of file
+  hide_project_coverage: false # set to false
```

Comment:
can refer to https://github.com/kubeedge/kubeedge/blob/master/.github/codecov.yml